src/window/building/figures.c

```diff
@@ -711,7 +712,15 @@ int window_building_handle_mouse_figure_list(const mouse *m, building_info_conte
             window_request_refresh();
         }
     }
+    if (c->terrain_type == TERRAIN_INFO_BRIDGE){
```

Comment:
```suggestion
    if (c->terrain_type == TERRAIN_INFO_BRIDGE) {
```

I see a spacing pattern :wink: